ui/app/styles/components/toolbar.scss

```diff
@@ -47,7 +47,7 @@
   @include from($mobile) {
     padding: 0 $spacing-xxs;
   }
-  // ARG TODO return, these are non-standard selectors
```

Comment:
They're still non-standard, but there's not a replacement. Potential for a future sparkle ticket to just not have scrollbar styling. Especially on the navbar???